feature-libs/order/components/order-details/order-detail-reorder/reorder-dialog/reorder-dialog.component.ts

```diff
@@ -40,6 +47,15 @@ export class ReorderDialogComponent {
   showDecisionPrompt$ = new BehaviorSubject(true);
   data$ = this.launchDialogService.data$;
 
+  @Optional() selectFocusUtility = inject(SelectFocusUtility, {
```

Comment:
It is good that `selectFocusUtility` and `elementRef` are optional injectors now. But we should include that `TODO` cleanup comment to also make them non-optional injections when we remove the `featureConfigService`.

Reply:
Added comments and a unit test to ensure the focus is set.